src/api/app/services/scm_status_reporter.rb

```diff
@@ -24,7 +24,12 @@ def call
                                          @state,
                                          status_options)
     end
-  rescue Octokit::Error, Octokit::InvalidRepository, Gitlab::Error::Error => e
+  rescue Octokit::InvalidRepository
+    package = Package.find_by(name: @event_payload[:package])
```

Comment:
This needs to be scoped on the project as well:

```suggestion
    package = Package.find_by_project_and_name(@event_payload[:project], @event_payload[:package])
```